qasc/README.md

```diff
@@ -0,0 +1,15 @@
+# QASC
+
+* [evaluator](evaluator/) is the program used by the AI2 Leaderboard to evaluate submitted predictions.
+* `data` have example prediction files
+
+## Example usage
+
+To evaluate dummy predictions (every question is predicted to be `A`) against the train dataset, run this:
+
+```
+% python3 evaluator/evaluator.py -qa data/train.jsonl -p data/train-predictions.csv -o metrics.json 
+
+% cat metrics.json
+{"accuracy": 0.12417014998770592}
+```
```

Comment:
Can you add an extra couple of lines here to mention the command user should run with their dev set predctions (and make sure this evaluator gives expected results on their predictions) ? If everything looks okay, then they should submit the test set predictions to the leaderboard.

Reply:
Sure. Have a look at the latest update.